init_scripts/init-user.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
this is a bad comment - comments should not describe *what* you do, comments should descry *why* you do it or things that could be not obvious about it.

The header section of a script should also include its name and details about its purpose, when, how and by whom it is executed.